docs/userguide/tables.md

```diff
@@ -171,8 +171,51 @@ Tablesaw supports inner and outer joins between tables.
 
 
 ## Add and remove rows
+###Adding New Row
 
+To add a new row into the table you simply need to call
```

Comment:
This is a very common pattern. I can not believe we were missing this. Thanks for adding!

Reply:
Thanks @ryancerf actually i would like to contribute more in the documentations, so i think it would be great if i could get some guidelines for the same.